uaclient/api/u/pro/security/cves/v1.py

```diff
@@ -282,141 +252,119 @@ def _post_process_vulnerability_info(
+                for related_usn in cve.get("related_usns", [])
+            ],
+            related_packages=cve.get("related_packages", []),
         )
+        for cve_name, cve in sorted(
+            vulnerabilities.get("vulnerabilities", {}).items(),
+            key=lambda v: v[0],
+        )
+        if cve_name not in blocked_cves
+    }
 
-    return CVEVulnerabilitiesResult(
+    return PackageVulnerabilitiesResult(
+        packages=packages,
         cves=cves,
         vulnerability_data_published_at=util.parse_rfc3339_date(
             vulnerability_data_published_at
         ),
-        apt_updated_at=(get_apt_cache_datetime()),
+        apt_updated_at=get_apt_cache_datetime(),
     )
 
 
 def _vulnerabilities(
     options: CVEVulnerabilitiesOptions,
     cfg: UAConfig,
-) -> CVEVulnerabilitiesResult:
+) -> PackageVulnerabilitiesResult:
     """
     This endpoint shows the CVE vulnerabilites in the system.
     By default, this API will only show fixable CVEs in the system.
     """
+
+    # By default we return all affected CVEs. If a user provides
+    # both options, we just switch to the default approach
+    if options.unfixable and options.fixable:
+        options.unfixable = False
+        options.fixable = False
+
     series = system.get_release_info().series
 
     cve_vulnerabilities_result = get_vulnerabilities(
         parser=CVEParser(),
         cfg=cfg,
         series=series,
     )
-    cve_vulnerabilities = cve_vulnerabilities_result.vulnerabilities_info.get(
-        "vulnerabilities", {}
-    )
+    cve_vulnerabilities = cve_vulnerabilities_result.vulnerabilities_info
 
     return _parse_vulnerabilities(
         options=options,
-        cve_vulnerabilities=cve_vulnerabilities,
+        vulnerabilities=cve_vulnerabilities,
         vulnerability_data_published_at=cve_vulnerabilities_result.vulnerability_data_published_at,  # noqa
     )
 
 
-def _vulnerabilities_with_applied_fixes_count(
-    options: CVEVulnerabilitiesOptions,
-    cfg: UAConfig,
-) -> Tuple[CVEVulnerabilitiesResult, Dict[str, Any]]:
-    """
-    This endpoint shows the CVE vulnerabilites in the system.
-    By default, this API will only show fixable CVEs in the system.
-    """
-    series = system.get_release_info().series
-
-    cve_vulnerabilities_result = get_vulnerabilities(
-        parser=CVEParser(),
-        cfg=cfg,
-        series=series,
-    )
-    cve_vulnerabilities = cve_vulnerabilities_result.vulnerabilities_info.get(
-        "vulnerabilities", {}
-    )
-
-    return (
-        _parse_vulnerabilities(
-            options=options,
-            cve_vulnerabilities=cve_vulnerabilities,
-            vulnerability_data_published_at=cve_vulnerabilities_result.vulnerability_data_published_at,  # noqa
-        ),
-        cve_vulnerabilities_result.vulnerabilities_info.get(
-            "applied_fixes_count", {}
-        ),
-    )
-
-
 endpoint = APIEndpoint(
     version="v1",
     name="CVEVulnerabilities",
```

Comment:
Same thing but down here
```suggestion
    name="CVEs",
```